internal/action/env_test.go

```diff
@@ -34,6 +34,9 @@ func TestEnv(t *testing.T) {
 		stdout = os.Stdout
 	}()
 
+	// Command-line would be: "gopass env foo env", where "foo" is an existing
+	// secret with value "secret". We expect to see the key/value in the output
+	// of the /usr/bin/env utility in the form "FOO=secret".
 	assert.NoError(t, act.Env(gptest.CliCtx(ctx, t, "foo", "env")))
-	assert.Contains(t, buf.String(), "secret")
+	assert.Contains(t, buf.String(), "FOO=secret\n")
```

Comment:
This is a very good refinement for the existing test.
One could even consider populating foo with a long, random password to absolutely ensure that the correct secret is displayed. But this is already much better than before.

Reply:
Thanks.
Yes, I wanted to use something more unique than "secret", but it looks like that "secret" is the default value used by the mocking system (the test itself doesn't mention it), so I left it as-is. If you point me to how to inject a more unique value, I am happy to adapt the code here.

Reply:
Maybe you want to leave a TODO for me to add that later?
Definitely ok as-is for this PR.

Reply:
added TODO